MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,231 @@
+Moves the view to week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 3 => WorkWeekView<br/><br/></td><td>
+Moves the view to work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 4 => MonthView<br/><br/></td><td>
+Moves the view to work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 5 => TimelineDayView<br/><br/></td><td>
+Moves the view to Timeline day view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 6 => TimelineWeekView<br/><br/></td><td>
+Moves the view to Timeline week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 7 => TimelineWorkWeekView<br/><br/></td><td>
+Moves the view to Timeline work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 8 => TimelineMonthView<br/><br/></td><td>
+Moves the view to Timeline month view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 9 => AgendaView<br/><br/></td><td>
+Moves the view to Agenda view.<br/><br/></td></tr>
+</table>
+
+N> There is a framework issue with unable to remove or modify  default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193).
+
+#### MAC and iOS
+
+<table>
+<tr>
+<td>
+Command + 1 => DayView<br/><br/></td><td>
+Moves the view to day view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 2 => WeekView<br/><br/></td><td>
+Moves the view to week view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 3 => WorkWeekView<br/><br/></td><td>
+Moves the view to work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 4 => MonthView<br/><br/></td><td>
+Moves the view to work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 5 => TimelineDayView<br/><br/></td><td>
+Moves the view to Timeline day view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 6 => TimelineWeekView<br/><br/></td><td>
+Moves the view to Timeline week view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 7 => TimelineWorkWeekView<br/><br/></td><td>
+Moves the view to Timeline work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 8 => TimelineMonthView<br/><br/></td><td>
+Moves the view to Timeline month view.<br/><br/></td></tr>
+<tr>
+<td>
+Command + 9 => AgendaView<br/><br/></td><td>
+Moves the view to Agenda view.<br/><br/></td></tr>
+</table>
+
+N> There is a framework issue with focus events not working in MAC and iOS and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10192).
+
+### Header
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td>Move to next focusable element in header.<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Move to previous focusable element in header.<br/><br/></td></tr>
+</table>
+
+N> You can also explore our [MAUI Scheduler example](https://github.com/syncfusion/maui-demos) to knows how to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.
```

Comment:
N> You can also explore our [MAUI Scheduler example](https://github.com/syncfusion/maui-demos) to know how to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.